calls/2024/2024-06-13_DFDL-WG-Call.md

```diff
@@ -0,0 +1,162 @@
+# OGF DFDL Working Group Call Agenda and Minutes 13 June 2024
+
+### Meeting Time
+11:00 - 12:00 EDT.US (UTC-4) 
+
+### Attendees
+TBD
+<!--
+* Mike Beckerle
+* Steve Hanson
+* Bradd Kadlecik
+* Liam O'Neill
+-->
+
+### IPR statement
+"I acknowledge that participation in this meeting is subject to the OGF Intellectual Property Policy"
+
+### Meeting closed
+TBD
+<!--
+11:56 EDT.US
+-->
+
+### Next regular call (Monthly on 2nd Thursday)
+
+Usual time would be: 2024-07-11 @ 11:00 EDT.US(UTC-4)
+
+Note: Mike Beckerle unavailable at that time. 
+
+
+## Agenda & Minutes
+
+1. Review of open action items (see dated comments below)
+
+2. Proposed new action item: Convert DFDL spec to XML (from MS Word) as way to create the first 
+   ISO formatted version, and get away from MS-Word. 
+
+3. 
+
+4. Administrivia
+  a.  Proposed: remove "deferred actions" from the call agenda/minutes to github tickets as place to park them
+
```

Comment:
Also, my mailman and mail archives links are now broken. 

- https://www.ogf.org/mailman/options/dfdl-wg
- https://www.ogf.org/pipermail/dfdl-wg/

The latter page is quite broken. Shows up as a forwarding link that can never be clicked.